### ISSUE TITLE
[steamshovel] save background color in session (Trac #1072)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1066]
The background color is not currently being saved into the session.

Migrated from https://code.icecube.wisc.edu/ticket/1072
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:10", 
    "description": "The background color is not currently being saved into the session.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335650323600", 
    "component": "combo core", 
    "summary": "[steamshovel] save background color in session", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-07-29T15:27:39", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150801T08:34:58: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 135758:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135758"
> made qstringlist, qcolor accessible from python, exposed backgroundColor from GLWidget, closes #1066
> ```


* hdembinski changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas commented:

> Milestone offline V15-12-00 deleted


* olivas changed milestone from "offline V15-12-00" to ""
